docs/quick-start.md

```diff
@@ -330,6 +331,10 @@ Note that Spark artifacts are tagged with a Scala version.
   <name>Simple Project</name>
   <packaging>jar</packaging>
   <version>1.0</version>
+  <properties>
```

Comment:
We're not giving the complete Maven config here, and Java 8 is required, so that's one of many pieces of config I think we assume the user will fill in. (Anyway, this is probably better specified by configuring the compiler plugin directly)?

But I agree with the two changes above. Any other bits of Java example with a similar problem?